File: FOODY/app/src/main/java/com/example/foody/done.java
package com.example.foody;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

public class done extends AppCompatActivity {
    Handler handler;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_done);
        handler=new Handler();
        handler.postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent intent=new Intent(done.this,map.class);
                startActivity(intent);
                finish();

            }
        },3000);
    }
}

File: app/src/main/java/com/example/foody/done.kt
package com.example.foody

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity

class done : AppCompatActivity() {
    var handler: Handler? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_done)
        handler = Handler()
        handler!!.postDelayed({
            val intent = Intent(this@done, map::class.java)
            startActivity(intent)
            finish()
        }, 3000)
    }
}